teamsupport/models.py

```diff
@@ -0,0 +1,66 @@
+from property_caching import cached_property
+
+
+class Ticket(object):
+    _data = None
+
+    def __init__(self, client, ticket_id=None, data=None):
+        self.client = client
+        if data is not None:
+            self._data = data
```

Comment:
If you move this out of the if block, you can get rid of the `_data = None` above.